src/lib/cli.rs

```diff
@@ -111,11 +115,11 @@ fn run(cli_args: CliArgs, global_config: &GlobalConfig) {
 
     let crate_name = envmnt::get_or("CARGO_MAKE_CRATE_NAME", "");
     if crate_name.len() > 0 {
-        info!("Project: {}", &crate_name);
+        debug!("Project: {}", &crate_name);
     }
-    info!("Build File: {}", &build_file);
```

Comment:
all these 3 are pretty important as i see it.
i moved other things to verbose, but these 3 are basic.

Reply:
I'd skip things like logging the cargo-make version on recursive calls then: when you have multiple Makefiles in a project and get this message many times, it makes it IMHO harder to actually parse the output in your head.